datasets/gov/vstt/lankytoju_monitoringas.csv

```diff
@@ -0,0 +1,13 @@
+id,dataset,resource,base,model,property,type,ref,source,prepare,level,access,uri,title,description
+,datasets/gov/vstt/lankytojai,,,,,,,,,,,,Saugomų teritorijų lankytojų duomenys,
+,,,,,,prefix,spinta,,,,,https://github.com/atviriduomenys/spinta/issues/,,
+,,,,,,,manifest,,,,,https://github.com/atviriduomenys/manifest/issues/,,
+,,,,,,,vadovas,,,,,https://atviriduomenys.readthedocs.io/,,
+,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,
+,,,,Monitoringas,,,vieta,,,2,,,Saugomų teritorijų lankytojų duomenys,
+,,,,,vieta,string,,,,4,open,,Lankoma vieta,
```

Comment:
```suggestion
,,,,,vieta,string,,,,2,open,,Lankoma vieta,
```
Mano nuomone duomenys yra denormalizuoti ir nenurodytas siejimas su kanoniniu duomenų rinkiniu: 
https://github.com/atviriduomenys/manifest/blob/1effdfe33bb744af2077cb5a2b09058a808274e8/datasets/gov/vstt/saugomu_ter_kadastras.csv
Pirminiu vertinimu duomenyse pateikiami to paties Saugomų teritorijų kadastro rinkinio skirtingų modelių duomenys: pvz. `/datasets/gov/vstt/teritorijos/Parkas`, `/datasets/gov/vstt/teritorijos/BiosferosRezervatas` ir pan.
Prašau pakartotinai įvertinti.
Brandos lygį, prašau įvertinti pagal rekomendacijas: 
https://atviriduomenys.readthedocs.io/dsa/ref.html#brandos-lygis
https://atviriduomenys.readthedocs.io/dsa/level.html#